webpack.config.development.js

```diff
@@ -50,23 +53,26 @@ module.exports = {
     ]
   },
   plugins: [
-    new webpack.NamedModulesPlugin(),
+    new webpack.optimize.OccurrenceOrderPlugin(),
     new webpack.HotModuleReplacementPlugin(),
+    new webpack.NamedModulesPlugin(),
     new webpack.NoEmitOnErrorsPlugin(),
     new HtmlWebpackPlugin({
-      template: "public/index.html"
-      // favicon: "public/favicon.ico"
+      template: "public/index.html",
+      favicon: "public/assets/img/favicon.ico"
     })
+    // insert chunks here, if you have any
     // new webpack.optimize.CommonsChunkPlugin({
     //   name: ["vendor"],
     //   minChunks: Infinity
     // })
   ],
   devServer: {
-    host: "localhost",
-    port: port,
     historyApiFallback: true,
+    hot: true,
     open: true,
```

Comment:
The browsers does not automatically open unless I use `webpack-dev-server`, other options??